packages/client/hmi-client/src/components/models/tera-reflexives-toolbar.vue

```diff
@@ -72,96 +71,120 @@ const stateId2NameMap = computed<{ [id: string]: string }>(() => {
 	return map;
 });
 
```

Comment:
Issue 1: when the user adds reflexives in Step 3, then navigates away from Step 3 (either going back, or hitting stratify) and then they return to Step 3, the menu that prompts the user to add reflexives disappeared, because the logic to populate it (in `populateReflexiveOptions`) is based on comparing two models (`modelToUpdate` and `modelToCompare`) and checking to see if there are any missing types between them. However, once you've added a reflexive, the types between base and strata models will be the same, so the prompts won't appear and you won't be able to edit your selections, e.g. add/remove additional reflexives. 

Solution 1: `typedModel`, which is the amr that is being updated, is now a copy of `modelToUpdate` (previously they referenced the same object), so that the reflexives added to `typedModel` don't affect the logic in `populateReflexiveOptions`

Issue 2: the user was unable to remove reflexives after adding them (e.g. if you made a mistake with your selection, you couldn't undo it)

Solution 2: to my knowledge, the MultiSelect component (used for the dropdown to select which states to add reflexives to) doesn't know about past selections or what the user has selected or deselected, it only knows about the current selected options. So, to allow the user to remove added reflexives when they uncheck an option, every time the selection is updated, I overwrite `typedModel` with the reflexives added based on the user's current selection. 

Issue 3: there can be multiple MultiSelect components, one for each reflexive type that you want to add (e.g. Infect, Recover) that would overwrite each other. 

Solution  3: combine all the selections made from each MultiSelect in the `addedReflexivesRows`, and apply them all at once in `updateStatesToAddReflexives`